1001-2000/LIT1693John.xml

```diff
@@ -81,34 +83,61 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+               <div type="textpart" subtype="incipit"><ab>ዝንቱሰ፡ ዮሐንስ፡ ፩፡ እም፲ወ፪፡ ሓዋርያት። ስማ፡ ለእሙ፡
+               ባውፍልያ፡ ወካዕበ፡ ተሰምየት፡ ማርያም። ወውእቱ፡ እምነገደ፡ ዛብሎን፡ ወእምነ፡ ቤተ፡ ሳይዳ፡ ጸሓፈ፡ ወንጌሎ፡ በልሳነ፡ ዮናናውያን፡
+               በ፰፡ ዓመት፡ እመንግሥተ፡ ኔሮን፡ ወልደ፡ አቅሎንድዮስ፡ ዘቀተሎሙ፡ ለጴጥሮስ፡ ወለጳውሎስ። በ፲ወ፫፡ ዓመተ፡ መንግሥቱ፡ እምድኅረ፡
+               ዕርገቱ፡ ለእግዚእነ፡ በ፴፡ ዓመት።</ab></div>
+               <div type="textpart" subtype="explicit"><ab>ወተብህለ፡ ከመ፡ ዮሐንስ፡ ኣወፈዮ፡ መጽሓፈ፡ አቡቀለምሲስ፡ ለፋጊር።</ab></div>
+            </div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="EusebianSectionsJohn">
+              <label>Statement about Eusebian Sections in the Gospel of John</label>
+              <div type="textpart" subtype="incipit"><ab>ወኍልቈ፡ ምዕራፈ፡ ወንጌሉ፡ ንኡሳን። ፪፻፡ ፴ወ፪፡</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወምዕራፋቲሁ፡ በልሳነ፡ ሮሜ። ፳</ab></div>
+            </div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="TituliJohn">
+              <label>Tituli of the Gospel of John</label>
+              <div type="textpart" subtype="incipit"><ab>፩፡ ተአምር፡ ዘቃና፡ ዘገሊላ። ፪፡ ዘከመ፡ ሰደደ፡ እምኰራብ። ፫፡ ኒቆዲሞስ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>፲፰፡ ዘከመ፡ ነሥኣ፡ ዮሴፍ፡ ሥጋሁ። ፲፱፡ ዘከመ፡ ዜነዉ፡ መላእክት፡ ትንሣኤ፡ ለአንስት።
+              ፳፡ በዊኦቱ፡ ኀበ፡ አርድእት፡ እንዘ፡ ዕፅው፡ ኃዋኅው።</ab></div>
+            </div>
             </div>
 
             <div type="textpart" subtype="chapter" xml:lang="gez" corresp="LIT2715John">
                <label>Bǝsrāta Yoḥannǝs</label>
+               <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="JohnGospelHeadline">
+               <label>Headline of the Gospel of John</label>
+               <ab><seg type="title inscriptio expanded">ብስራተ፡ አብ፡ ቅዱስ፡ ትሩፍ፡ ረድእ፡ ተናገሪ፡ በመለኮት፡ ዮሓንስ፡ ወልደ፡
+                ዘብዴዎስ፡ ፍቁረ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ።</seg> ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ዘመለኮት፡ ለዓመለ፡ ዓለም፡ አሜን፡</ab>
+            </div>
+            <div type="textpart" subtype="coretext" xml:lang="gez" xml:id="JohnGospel">
+               <label>Gospel of John</label>
+               <div type="textpart" subtype="incipit"><ab>ቀዳሚሁ፡ ውእቱ፡ ቃል፡ ወውእቱ፡ ቃል፡ ኀበ፡ እግዚኣብሔር፡ ሀለወ፡ ወእግዚኣብሔር፡ ውእቱ፡
+                ቃል፡ ዝንቱ፡ እምቀዲሙ፡ ሀሎ፡ ኀበ፡ እግዚኣብሔር። ኵሉ፡ ቦቱ፡ ኮነ፡ ወዘእንበሌሁሰ፡ አልቦ፡ ዘኮነ፡</ab></div>
+               <div type="textpart" subtype="explicit"><ab>ወሶበ፡ ተጽሕፈ፡ ኵሉ፡ በበ፡ ኣሓዱ፡ እምኢያግመረ፡ ጥቀ፡ መጻሕፍቲሁ፡ ዘተጽሕፈ፡ አሜን፡ ወአሜን፡</ab></div>
             </div>
+            <div type="textpart" subtype="colophon" xml:lang="gez" xml:id="Postscript">
+              <label>Colophon of the Gospel of John</label>
+              <div type="textpart" subtype="incipit"><ab>መልአ፡ <seg type="title subscriptio expanded">ስብከተ፡ ዮሓንስ፡ ወልደ፡ ዘብዴዎስ፡ ሓዋርያ፡
+              ፩፡ እም፲ወ፪፡</seg>እንተ፡ ጸሓፎ፡ በዮናኒ፡ ለሰብኣ፡ ሀገረ፡ ኤፌሶን።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>በ፰፡ ዓመተ፡ መንግሥቱ፡ ለጢባርዮስ፡ ቄሳር። ጸሎቱ፡ ወበረከቱ፡ ለቅዱስ፡ ዮሓንስ፡ የሀሉ፡ ምስለ፡ ገብሩ፡
+              ዘመለኮት፡ ወፍቁሩ፡ በእደ፡ ማርያም፡ ወጻሓፊሃ፡ ኃጥእ፡ ገብረ፡ አምላክ፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን።</ab></div>
```

Comment:
Proper names ዘመለኮት፡ and ገብረ፡ አምላክ፡ could be replaced with `<gap reason="ellipsis" resp="Abr"/>`